src/main/java/core/basesyntax/service/impl/DataParserImpl.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.DataParser;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataParserImpl implements DataParser {
+    private static final String COMMA = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_NAME_INDEX = 1;
+    private static final int FRUIT_QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> parseAll(List<String> data) {
+        List<FruitTransaction> fruitTransactionList = new ArrayList<>();
+
+        for (String line : data) {
+            if (!line.equals("type,fruit,quantity")) {
```

Comment:
Actually, I suggest just skipping first element of data, in future, we could change the header line, or maybe some whitespace will be added to header and this code will not work anymore

Reply:
Not resolved